documentation/tutorial.md

```diff
@@ -774,7 +774,7 @@ After `ssh`ing into `mongodb/0`, we are now in the unit that is hosting Charmed
 
 As before, we will connect to MongoDB via the saved MongoDB URI. Connect using the saved URI and the following TLS options:
 ```shell
-mongosh mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REPL_SET_NAME --tls --tlsCAFile /etc/mongodb/external-ca.crt --tlsCertificateKeyFile /etc/mongodb/external-cert.pem
+charmed-mongodb.mongo mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REPL_SET_NAME --tls --tlsCAFile /etc/mongodb/external-ca.crt --tlsCertificateKeyFile /etc/mongodb/external-cert.pem
```

Comment:
Similar to the other comment, should these paths be updated?  The snap shouldnt be able to access that directory.

Reply:
yes! great eyes thank you Jarred :)